lib/rubocop/cop/rspec/factory_bot/create_list.rb

```diff
@@ -213,6 +216,14 @@ def call_replacement(node)
               replacement
             end
 
+            def count_from(node)
+              if node.receiver.int_type?
+                node.receiver
+              else
+                node.send_node.first_argument
+              end.source
```

Comment:
Can we give a name to the thing we call `source` on? That might allow me to think a bit less about what the method does 😅 
```suggestion
              foo = if node.receiver.int_type?
                node.receiver
              else
                node.send_node.first_argument
              end
              foo.source
```

Reply:
OK, I'll use the name `count_node` for this variable 👍 

```ruby
  3.times
# ^ this node

  Array.new(3)
          # ^ or this node
```